skbio/metadata/_intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
That looks fine, though I thought there were changes made to the original bx-python code? (not sure what they were, but I think @mortonjt made changes in one of the previous PRs)

Minor change:

`license/bx_python.txt` -> `licenses/bx_python.txt`

Can you also add this comment to the bx-python test code (also noting any changes)?